pkg/ui/static/js/graph_template.handlebar

```diff
@@ -115,7 +115,7 @@
                         <div class="prometheus_input_group pull-left">
                            <select name="max_source_resolution_input" title="Downsampling option" id="max_source_resolution_input{{id}}">
                               <option value="">Auto downsampling</option>
-                              <option selected="selected" value="0s">Only raw data</option>
+                              <option value="0s">Only Raw data</option>
```

Comment:
Why did you take off `selected` here and converted it into `Raw`? It is changing the defaults AFAICT :(

Reply:
Converted Raw back, selected is still off, but now the code select "0s" if max_source_resolution is undefined. This keeps the default comportment to "Only raw data".